<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AOPlot</class>
 <widget class="QWidget" name="AOPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="plotSomething">
       <property name="text">
        <string>Plot 1d</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plotSomething2d">
       <property name="text">
        <string>Plot 2d</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
